src/elements/common/header/__tests__/Header.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
Think we're trying to switch away from enzyme, can we use testing-library instead